src/main/java/de/orchound/gameoflife/GameOfLifeApplet.java

```diff
@@ -194,8 +198,15 @@ private void resetBoard() {
 	}
 
 	private void randomize() {
-		board.randomize();
+		randomizeBoard();
 		boardRenderer.update();
 		gameTimeAccumulator = 0;
 	}
+
+	private void randomizeBoard() {
+		for (int i = 0; i < board.target.length; i++) {
+			board.target[i] = random.nextBoolean();
+		}
+		System.arraycopy(board.target, 0, board.initial, 0, board.target.length);
```

Comment:
I don't like the exposing of `initial`. Maybe we should provide a method like `makeCurrentStateInitial()` to set `initial` internally?